features/networking/ovn.feature

```diff
@@ -573,3 +573,62 @@ Feature: OVN related networking scenarios
       | ping | -c | 30 | -i | 0.2 | <%= cb.pod2_ip%> |
     Then the step should succeed
     And the output should contain "0% packet loss"
+  
+  # @author anusaxen@redhat.com
+  # @case_id OCP-46285
+  @admin
+  @destructive
+  @4.10 @4.9 @4.8
+  @network-ovnkubernetes
+  @vsphere-ipi
+  Scenario: Logical Router Policies and Annotations for a given node should be current
+  Given the env is using "OVNKubernetes" networkType
```

Comment:
@anuragthehatter seems the indentation for the whole steps for this case is incorrect.

Reply:
Thats right. Seems like my auto indent is not working fine post fedora upgrade. Thanks. Done for now